main/client/src/mill/main/client/MillClientMain.java

```diff
@@ -11,20 +11,8 @@
 
 public class MillClientMain {
     static void initServer(String lockBase, boolean setJnaNoSys) throws IOException,URISyntaxException{
-        ArrayList<String> selfJars = new ArrayList<String>();
-        ClassLoader current = MillClientMain.class.getClassLoader();
-        while(current != null){
-            if (current instanceof java.net.URLClassLoader) {
-                URL[] urls = ((java.net.URLClassLoader) current).getURLs();
-                for (URL url: urls) {
-                    selfJars.add(new File(url.toURI()).getCanonicalPath());
-                }
-            }
-            current = current.getParent();
-        }
-        if (Util.isJava9OrAbove) {
```

Comment:
@robby-phd do you remember what this was for, or why we need it?

Reply:
It was always there [before](https://github.com/lihaoyi/mill/blob/122122729d8567e389f434cfb1c074d3119b0088/clientserver/src/mill/clientserver/Client.java#L14-L25) I worked on changing mill client; I assume it's to propagate classpath to the spawned server.

Reply:
@robby-phd how about the `System.getProperty("java.class.path")` logic? It looks like you added it in https://github.com/lihaoyi/mill/commit/a9d4eea6f5b81cdbef84724e612bf07954e2673c#diff-00b5b77d5f332c03ac7aa4d9ac51c941R29